<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0" rowspan="3">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="0">
        <widget class="QCheckBox" name="savingCB">
         <property name="text">
          <string>Zapisz współrzędne krzywej</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="saveImageCB">
         <property name="text">
          <string>Zapisz obraz krzywej</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Wsp. X:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="xLE"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Wsp. y:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="yLE"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Waga:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="wLE"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="resultL">
         <property name="text">
          <string>Jakość dodanego punktu</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="result2L">
         <property name="text">
          <string>Dodajesz nową krzywą</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Dodaj Punkt</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="drawButton">
         <property name="text">
          <string>Rysuj</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Zamknij</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="openPB">
         <property name="text">
          <string>Odczytaj współrzędne z pliku</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>czerwony</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>zielony</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>niebieski</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>czarny</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Współrzędne krzywej odczytane z pliku:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="textEdit">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>INSTRUKCJA OBSŁUGI PROGRAMU + O PROGRAMIE</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
